signalfx-tracing/signalfx-nodejs-tracing/mysql/server.js

```diff
@@ -0,0 +1,36 @@
+// The core of the auto-instrumentation process is made in the
+// tracer.js module.  It consists of a single import and invocation
+// of the 'signalfx-tracing' module.
+const tracer = require('./deedScheduler/tracer');
+
+const Koa = require('koa');
+const logger = require('koa-logger');
+const scheduler = require('./deedScheduler');
+const server = new Koa();
+const router = require('koa-router')();
+const port = scheduler.config.serverPort;
+
+const version = scheduler.config.mysqlVersion;
+
+let sqlVersion;
+
+if (Number(version) === 1) {
+  sqlVersion = 'MySQL';
+} else {
+  sqlVersion = 'MySQL2';
+};
```

Comment:
I'd move this to the config as something like `mysqlLib`.  Also, this isn't the db version but is the driver instead.